core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -997,110 +588,129 @@ fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Decorative): Color {
-        primary = backgroundColorTokens.bgBrandPrimaryLight,
-        onPrimary = contentColorTokens.contentDefaultLight,
-        primaryContainer = backgroundColorTokens.bgPrimaryLight,
-        onPrimaryContainer = contentColorTokens.contentDefaultLight,
-        inversePrimary = backgroundColorTokens.bgBrandPrimaryDark,
-        secondary = backgroundColorTokens.bgSecondaryLight,
-        onSecondary = elevationColorTokens.elevationDragOnBgSecondaryLight,
-        secondaryContainer = backgroundColorTokens.bgEmphasizedLight,
-        onSecondaryContainer = contentColorTokens.contentDefaultDark,
-        tertiary = backgroundColorTokens.bgTertiaryLight,
-        onTertiary = contentColorTokens.contentDefaultDark,
-        tertiaryContainer = decorativeColorTokens.decorativeBrandTertiaryLight,
-        onTertiaryContainer = contentColorTokens.contentDefaultOnBgEmphasizedLight,
-        background = backgroundColorTokens.bgPrimaryLight,
-        onBackground = contentColorTokens.contentDefaultLight,
-        surface = elevationColorTokens.elevationRaisedLight,
-        onSurface = contentColorTokens.contentDefaultLight,
-        surfaceVariant = elevationColorTokens.elevationModalLight,
-        onSurfaceVariant = contentColorTokens.contentDefaultLight,
-        surfaceTint = elevationColorTokens.elevationModalLight,
-        inverseSurface = elevationColorTokens.elevationRaisedDark,
-        inverseOnSurface = contentColorTokens.contentDefaultDark,
-        error = backgroundColorTokens.bgStatusNegativeMutedLight,
-        onError = contentColorTokens.contentOnActionNegativeLight,
-        errorContainer = backgroundColorTokens.bgStatusNegativeEmphasizedLight,
-        onErrorContainer = contentColorTokens.contentOnActionDisabledOnBgEmphasizedLight,
-        outline = borderColorTokens.borderBrandPrimaryLight,
-        outlineVariant = borderColorTokens.borderBrandPrimaryLight,
-        scrim = contentColorTokens.contentBrandPrimaryLight,
-    )
+//TODO Material colors must be able to be overridden by a theme
+val materialLightColorScheme: ColorScheme
+    get() = with(OudsAndroidColorLightTokens) {
+        lightColorScheme(
+            primary = primary,
+            onPrimary = onPrimary,
+            primaryContainer = primaryContainer,
+            onPrimaryContainer = onPrimaryContainer,
+            inversePrimary = inversePrimary,
+            secondary = secondary,
+            onSecondary = onSecondary,
+            secondaryContainer = secondaryContainer,
+            onSecondaryContainer = onSecondaryContainer,
+            tertiary = tertiary,
+            onTertiary = onTertiary,
+            tertiaryContainer = tertiaryContainer,
+            onTertiaryContainer = onTertiaryContainer,
+            background = background,
+            onBackground = onBackground,
+            surface = surface,
+            onSurface = onSurface,
+            surfaceDim = surfaceDim,
+            surfaceVariant = surface,
```

Comment:
Maybe we should add missing `surfaceVariant` and `surfaceTint` now in `OudsAndroidColorXxxTokens` because there is a probability that we will forget to update the mapping here once these files are generated by Tokenator.